CMake/cdat_modules_extra/python_configure_step.cmake.in

```diff
@@ -29,7 +29,7 @@ if(APPLE)
 endif()
 
 if(APPLE)
-  set(library_param --prefix=@CMAKE_INSTALL_PREFIX@ --enable-framework=@CMAKE_INSTALL_PREFIX@/Library/Frameworks)
+  set(library_param --prefix=@CMAKE_INSTALL_PREFIX@ --with-system-expat --enable-framework=@CMAKE_INSTALL_PREFIX@/Library/Frameworks)
```

Comment:
The problem here is that the `Python 2.7.10` tarball ships with `expat` (and `pyexpat`) and attempts to build this internally within the `python` build.. I have no idea why this (and some other modules) fails within the `cmake` build statement, whereas when `python 2.7.10` is built **before** the `cmake` build step, it completes successfully.. See further down in the log linked directly above..